sources/packages/api/src/route-controllers/application/application.controller.ts

```diff
@@ -66,8 +68,17 @@ export class ApplicationController extends BaseController {
     @Body() payload: CreateApplicationDto,
     @UserToken() userToken: IUserToken,
   ): Promise<number> {
+    const programYear = await this.programYearService.getProgramYear(
```

Comment:
Thanks for renaming the method, but could we please use getActiveProgramYear instead?